### PR TITLE
feat: fix long time between mqtt message and readable message in dashboard

### LINKED ISSUE #660: Long time between MQTT message and readable message in dashboard (>30 seconds)
For an impressive impression, the time between the arrival of a message and its display in the dashboard should be <<5 sec

Please contact @kruegeralex  to get access to the test environment.

### PR DESCRIPTION
**Pull request by Scarjit**
Attempts to resolve #660 by introducing exponential backoff time.
See also #671 & #689

### CODE REVIEW

File: golang/go.sum [hunk 1]
```diff
@@ -189,6 +189,8 @@ github.com/prometheus/procfs v0.6.0 h1:mxy4L2jP6qMonqmq+aTtOx1ifVWUgG/TAmntgbh3x
 github.com/prometheus/procfs v0.6.0/go.mod h1:cz+aTbrPOrUb4q7XlbU9ygM+/jj0fzG6c1xBZuNvfVA=
 github.com/rung/go-safecast v1.0.1 h1:7rkt2qO4JGdOkWKdPEBFLaEwQy20y0IhhWJNFxmH0p0=
 github.com/rung/go-safecast v1.0.1/go.mod h1:dzUcUS2UMtbfVc7w6mx/Ur3UYcpXEZC+WilISksJ4P8=
+github.com/sbinet/go-gnuplot v0.0.0-20130514120836-9167d8eb1ac4 h1:DjPYuV8FWDg57bkdJ2DQUB6PMQJUapXdJ1uCvPC7XT0=
```
Comment (JeremyTheocharis) [inline golang/go.sum:192]: Do we really need it? I do not see it referenced in the code

Comment (JeremyTheocharis) [reply golang/go.sum:192]: ahh, it was used in an uncommented plot

File: golang/internal/exponential_backoff_test.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+		btime := GetBackoffTime(i, 1*time.Nanosecond, max)
+		if btime < 0 || btime > max {
+			t.Logf("BTime: %s", btime)
+			t.Logf("I: %d", i)
+			t.FailNow()
+		}
+	}
+}
+
+func TestPlotBackoffTime(t *testing.T) {
```
Comment (JeremyTheocharis) [inline golang/internal/exponential_backoff_test.go:20]: should probably remove it

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]
Only a smaller change regarding dependencies (open to discuss whether it makes sense).

I do not see any larger issues occuring from this PR, so testing it on staging should be fine (no direct testing for this PR)

File: golang/go.sum (hunk 1 above)
Comment (Scarjit) [reply golang/go.sum:192]: Resolved in https://github.com/united-manufacturing-hub/united-manufacturing-hub/pull/690/commits/ba6e7cf265033bfe0530b893eac8bc5d73ae0062

File: golang/internal/exponential_backoff_test.go (hunk 2 above)
Comment (Scarjit) [reply golang/internal/exponential_backoff_test.go]: Resolved in https://github.com/united-manufacturing-hub/united-manufacturing-hub/pull/690/commits/ba6e7cf265033bfe0530b893eac8bc5d73ae0062

**Review by JeremyTheocharis — APPROVED** [review]

**Pull request merged**